android-sdk/android/src/main/AndroidManifest.xml

```diff
@@ -5,26 +5,31 @@
         <intent>
             <action android:name="android.support.customtabs.action.CustomTabsService" />
         </intent>
+        <package android:name="com.tencent.mm" />
     </queries>
 
     <application>
-        <activity android:name=".auth.webview.WebViewAuthActivity"
+        <activity android:name=".auth.logto.LogtoWebViewAuthActivity"
             android:exported="true"
             android:launchMode="singleTop"/>
 
-        <activity android:name=".auth.CallbackUriActivity"
-                  android:exported="true">
+        <activity android:name=".auth.social.web.WebSocialResultActivity"
+            android:exported="true">
             <intent-filter>
                 <action android:name="android.intent.action.VIEW" />
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
-                <data android:scheme="@string/logto_redirect_scheme" />
+                <data android:scheme="${applicationId}.logto-callback-web" />
             </intent-filter>
+        </activity>
+
+        <activity android:name="com.alipay.sdk.app.AlipayResultActivity"
+            android:exported="true">
             <intent-filter>
-                <action android:name="android.intent.action.VIEW" />
-                <category android:name="android.intent.category.DEFAULT" />
-                <category android:name="android.intent.category.BROWSABLE" />
-                <data android:scheme="logto-callback" />
+                <action android:name="android.intent.action.VIEW"/>
+                <category android:name="android.intent.category.DEFAULT"/>
+                <category android:name="android.intent.category.BROWSABLE"/>
+                <data android:scheme="${applicationId}.logto-callback-alipay"/>
```

Comment:
The code style seems not to be consistent.
Better separate a PR to format code style.

See the code above
<img width="557" alt="image" src="https://user-images.githubusercontent.com/10594507/163754488-080311f2-fc54-446d-9329-82f5b017ba81.png">

Reply:
updated